src/DynamoCoreWpf/ViewModels/Preview/WatchViewModel.cs

```diff
@@ -256,11 +257,27 @@ public void CountNumberOfItems()
         }
 
         /// <summary>
-        /// Count the list levels of each list 
+        /// Set the list levels of each list 
         /// </summary>
         public void CountLevels()
         {
-            Levels = maxListLevel > 0 ? Enumerable.Range(1, maxListLevel).Reverse().Select(x => x).ToList() : Enumerable.Empty<int>();
+            Levels = maxListLevel > 0 ? Enumerable.Range(1, maxListLevel).Reverse().Select(x => new Level() { Levels = x, LeftMargin = 0 }).ToList() : Enumerable.Empty<Level>();
         }
     }
+
+    public class Level
+    {
+        private int leftMargin;
+        public int Levels { get; set; }
+
+        public int LeftMargin
+        {
+            get { return leftMargin; }
+            set
+            {
+                leftMargin = this.Levels == 1 ? leftMargin = 11 : leftMargin = value;
```

Comment:
Can we use a converter, say `LevelToMargin` converter, instead of putting the logic in this Level class?